src/app/custom-components/event-location/event-location.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component } from "@angular/core";
+import { IMapOptions, IMarkerIconInfo, ILatLong } from "angular-maps";
+
+const LATITUDE: number = -17.41809191207568;
+const LONGITUDE: number = -66.12963527113635;
+@Component({
+  selector: "sw-event-location",
+  templateUrl: "./event-location.component.html",
+  styleUrls: ["./event-location.component.scss"]
```

Comment:
You are using double quotes instead of single quotes. You have to configure Prettier in order to use single quotes.